cfme/tests/automate/test_service_dialog.py

```diff
@@ -59,12 +58,7 @@ def test_service_dialog_duplicate_name(appliance, request):
     label = 'duplicate_' + fauxfactory.gen_alphanumeric()
     dialog = create_dialog(appliance, element_data, label=label)
     request.addfinalizer(dialog.delete_if_exists)
-    error_message = version.pick({
-        '5.8': 'Validation failed: Label is not unique within region 0',
-        '5.9': 'There was an error editing this dialog: '
-               'Failed to create a new dialog - Validation failed: '
-               'Name is not unique within region 0'})
-    with error.expected(error_message):
+    with pytest.raises(AssertionError):
```

Comment:
@sshveta This is sufficient - but consider actually asserting the flash message here. You know what view is expected, and can create that view object to assert against its flash message widget.